doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+- If using the autoencoder loss, then we need access to the original dataset
+
+### Local Feature Attribution
+
+Local feature attribution asks how each feature in a given instance contributes to its prediction. In the case of an image, this would highlight those pixels that make the model provide the output it does. Note that this differs subtly from Local Necessary Features, which find the minimum subset of features required to give a prediction. Local feature attribution instead assigns a score to each feature.
+
+__TODO__:
+- picture showing above.
+
+A good example use of local feature attribution is to detect that a classifier trained on images is focusing on the correct features of an image to infer the class. Suppose you have a model trained to classify breeds of dogs. You want to check that it focuses on the correct features of the dog in making its prediction. Suppose you compute the feature attribution of a picture of a husky and discover that the model is only focusing on the snowy backdrop to the husky, then you know two things. All the images of huskies in your dataset overwhelmingly have snowy backdrops, and also that the model will fail to generalize. It will potentially incorrectly classify other dog breeds with snowy backdrops as huskies and fail to recognize huskies that aren't in snowy locations.
+
+Each of the following methods defines local feature attribution slightly differently. In both, however, we assign attribution values to each feature to indicate how significant those features were in making the model prediction.
+
+Let $f:\mathbb{R}^n \rightarrow \mathbb{R}$. $f$ might be a regression, a single component of a multi regression or a probability of a class in a classification model. If $x=(x_1,... ,x_n) \in \mathbb{R}^n$ then an attribution of the prediction at input $x$ is a vector $a=(a_1,... ,a_n) \in \mathbb{R}^n$ where $a_i$ is the contribution of $x_i$ to the prediction $f(x)$.
+
+The attribution values should satisfy specific properties:
+
+1. Efficiency/Completeness: The sum of attributions equals the difference between the prediction and the baseline/average. We're interested in understanding the difference each feature value makes in a prediction compared to some uninformative baseline.
+2. Symmetry: If the model behaves the same after swapping two variables $x$ and $y$, then $x$ and $y$ have equal attribution. If this weren't the case, we would be biasing the attribution towards certain features over other ones.
+3. Dummy/Sensitivity: If a variable does not change the output of the model, then it should have attribution 0. If this were not the case, we'd be assigning value to a feature that provides no information.
+4. Additivity/Linearity: The attribution for a feature $x_i$ of a linear composition of two models $f_1$ and $f_2$ given by $c_1 f_1 + c_2 f_2$ is $c_1 a_{1, i} + c_2 a_{2, i}$ where $a_{1, i}$ and $a_{2, i}$ is the attribution for $x_1$ and $f_1$ and $f_2$ respectively.
```

Comment:
I would bold the names of the properties for emphasis, e.g. **Efficiency/Completeness**